charts/victoria-metrics-agent/templates/service.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: Service
+metadata:
+{{- if .Values.service.annotations }}
+  annotations:
+{{ toYaml .Values.service.annotations | indent 4}}
+{{- end }}
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+{{- if .Values.service.labels }}
+{{ toYaml .Values.service.labels | indent 4}}
+{{- end }}
+  name: {{ template "chart.fullname" . }}
```

Comment:
Let's create a separate function to generate a service name (and use it in ingress manifest also). It will help to specify indirect service name.